package.json

```diff
@@ -157,7 +157,7 @@
     "pm2-runtime": "./bin/pm2-runtime"
   },
   "dependencies": {
-    "async": "1.5",
+    "async": "^2.4.1",
```

Comment:
Agreed, mostly. I'd advise to check the changelog though just to be sure we don't miss anything :).